### ISSUE TITLE
Cannot launch basic Interface in `Blocks-dev` branch

### ISSUE THREAD

**Issue by abidlabs** [gradio-app/gradio#847]
### Describe the bug

When I try to launch a simple `Interface` from the `Blocks-dev` branch, I get a blank white page with the following error in the console:

```
Uncaught (in promise) ReferenceError: input_component_map is not defined
    at Ge (index.a875c593.js:1:3028)
    at Ne (index.a875c593.js:1:4277)
    at Mt (index.a875c593.js:3:1579)
    at Jr (vendor.cb9b505c.js:1:7101)
    at new Bt (index.a875c593.js:3:6897)
    at zl (index.a875c593.js:8:742)
    at Jr (vendor.cb9b505c.js:1:7101)
    at new Gl (index.a875c593.js:8:3590)
    at window.launchGradio (index.a875c593.js:8:18055)
    at index.a875c593.js:8:18366
```

### Reproduction

Check out the `Blocks-dev` branch and try to create a simple Interface like:

```python
import gradio as gr
gr.Interface(lambda x:x, gr.components.Textbox(), gr.components.Textbox()).launch()
```

or 

```python
import gradio as gr
gr.Interface(lambda x:x, "textbox", "textbox").launch()
```

### System Info

```shell
Edge browser, Windows 11
```

### Severity

blocker

**Comment by aliabid94** [comment]
Seems like frontend in Blocks-Dev is in a broken state right now, the code to choose between input and output component has not been written yet, and the old code (using the in/output_component_map) has been removed.

**Comment by abidlabs** [comment]
Issue will be fixed by #823
